commands/global/button_scr.h

```diff
@@ -1,2 +1,12 @@
+//#include "pirate/button.h"
+enum button_codes;
 void button_scr_handler(struct command_result *res);
-bool button_exec(void);
\ No newline at end of file
+bool button_exec(enum button_codes button_code);
+
+typedef struct {

```

Comment:
Do these three variable need to be global/exposed to other code? I believe they're only used internal to button_scr.c?

Reply:
the struct plus 
extern button_press_type_t button_press_types[];
extern const uint8_t num_button_press_types;